README.md

```diff
@@ -88,3 +88,8 @@ php.ini
 ## Developer documentation
 
 See the [Magento Commerce Cloud Guide](http://devdocs.magento.com/guides/v2.2/cloud/bk-cloud.html).
+
+## License
+Each Magento source file included in this distribution is licensed under OSL-3.0 license.
+
+Please see [LICENSE.txt](https://github.com/magento/ece-tools/blob/develop/LICENSE.txt) for the full text of the [Open Software License v. 3.0 (OSL-3.0)](http://opensource.org/licenses/osl-3.0.php).
```

Comment:
Link is going to ece-tools -- should it be going to this repo?

Reply:
Updated.